packages/installer/cos.sh

```diff
@@ -405,7 +405,6 @@ get_iso()
 get_image()
 {
     if [ -n "$UPGRADE_IMAGE" ]; then
-        part_probe
```

Comment:
@mudler any idea why was this here? This did nothing as there was no `$dev` passed to the call, but didnt fail due to the `|| true` in the function.

Reply:
looks like a leftover, `blkid_probe` was actually probably meant here